### PR TITLE
Add select function to p2p connection - Closes#931

### LINKED ISSUE #931: Add selectForConnection function to P2P
### Description
Add `selectForConnection` should look at the current lists of peers and choose, which nodes to connect to.
In the phase 1, it should just return all of the peers

### PR DESCRIPTION
**Pull request by ishantiw**
### Description 

Implement selectForConnection to make a selection and connect with peers. This implementation covers phase 1 which simply returns all the known peers.

### Review checklist

* The PR resolves #931
* All new code is covered with unit tests
* All new code was formatted with Prettier
* Linting passes
* Tests pass
* Commit messages follow the [commit guidelines](CONTRIBUTING.md#git-commit-messages)
* Documentation has been added/updated

### CODE REVIEW

File: packages/lisk-p2p/test/integration/p2p.ts [hunk 1]
```diff
@@ -1,40 +1,40 @@
-import { P2P } from '../../src/index';
+// import { P2P } from '../../src/index';
 
-describe('Integration tests for P2P library', () => {
-	const NETWORK_PEER_COUNT = 10;
-	let blockchainP2PList: ReadonlyArray<P2P> = [];
+// describe('Integration tests for P2P library', () => {
```
Comment (shuse2) [inline packages/lisk-p2p/test/integration/p2p.ts]: instead of commenting out, u can use `skip`

**Review by shuse2 — CHANGES_REQUESTED** [review]
also, skipping integration test can be on different PR?

File: packages/lisk-p2p/test/integration/p2p.ts (hunk 1 above)
Comment (ishantiw) [reply packages/lisk-p2p/test/integration/p2p.ts]: yeah I forgot that I can use `skip`, also, I will take it out from here and do it in a separate PR

**Comment by ishantiw** [comment]
@shuse2 Removed integration test change and ready for review

**Review by shuse2 — APPROVED** [review]

**Review by jondubois — APPROVED** [review]

**Pull request merged**